styles/components/Footer.module.scss

```diff
@@ -0,0 +1,70 @@
+.Footer {
+  @apply grid grid-cols-2 gap-4 p-4;
+  @apply bg-color-3;
+
+  a {
+    @apply text-color-2;
+    @apply underline;
+
+    &:visited,
+    &:active {
+      @apply text-color-2;
+    }
+
+    &:hover {
+      @apply text-hover-1;
+    }
+  }
+}
+
+.top {
+  @apply col-span-2;
+  @apply text-center;
+  @apply text-color-2;
+  @apply my-4;
+}
+
+.left {
+  @apply flex gap-4 items-center;
+
+  a {
+    @apply text-color-2;
+    @apply underline;
+
+    &:hover {
+      @apply text-hover-1;
+    }
+  }
+
+  .logo {
```

Comment:
Shouldn't the logo element be handling a lot of this styling itself?

Reply:
removed the stuff that it already handles now